external/storm-kafka/src/jvm/storm/kafka/bolt/KafkaBolt.java

```diff
@@ -53,10 +55,13 @@
     public static final String TOPIC = "topic";
     public static final String KAFKA_BROKER_PROPERTIES = "kafka.broker.properties";
 
-    private Producer<K, V> producer;
+    private KafkaProducer<K, V> producer;
     private OutputCollector collector;
     private TupleToKafkaMapper<K,V> mapper;
     private KafkaTopicSelector topicSelector;
+    /* With default setting for fireAndForget and async, the callback is called when sending succeed */
+    private boolean fireAndForget = false;  /* by setting as true, each send immediately gets acked by collector */
```

Comment:
lets clarify this comment saying it doesn't wait at all for kafka to ack.  ack setting doesn't matter if this is used.